contents/blog/posthog-alternatives.md

```diff
@@ -0,0 +1,738 @@
+        	<td className="text-center">Manage feature access and rollouts</td>
+    	</tr>
+    	<tr>
+        	<td>A/B testing</td>
+        	<td className="text-center"><span className="text-green text-lg">✔</span></td> 
+        	<td className="text-center"><span className="text-green text-lg">✔</span></td>
+        	<td className="text-center">CRO and test in production</td>
+    	</tr>
+    	<tr>
+        	<td>Group analytics</td>
+        	<td className="text-center"><span className="text-red text-lg">✖</span></td> 
+        	<td className="text-center"><span className="text-green text-lg">✔</span></td>
+        	<td className="text-center">Analyze groups (e.g. companies and teams)</td>
+    	</tr>
+    	<tr>
+        	<td>SQL access</td>
+        	<td className="text-center"><span className="text-red text-lg">✖</span></td> 
+        	<td className="text-center"><span className="text-green text-lg">✔</span></td>
+        	<td className="text-center">Write your own queries and filters</td>
+    	</tr>
+    	<tr>
+       	<td>Transparent pricing</td>
+      		<td className="text-center"><span className="text-green text-lg">✔</span></td>
+       	<td className="text-center"><span className="text-green text-lg">✔</span></td>
+        	<td className="text-center">Know exactly what you'll pay</td>
+    	</tr>
+    	<tr>
+       	<td>Billing limits</td>
+      		<td className="text-center"><span className="text-red text-lg">✖</span></td>
+       	<td className="text-center"><span className="text-green text-lg">✔</span></td>
+        	<td className="text-center">Prevent surprise bills</td>
+    	</tr>
+    	<tr>
+        	<td>Open source</td>
+        	<td className="text-center"><span className="text-red text-lg">✖</span></td>
+        	<td className="text-center"><span className="text-green text-lg">✔</span></td>
+        	<td className="text-center">Open core, build your own integrations</td>
+   		</tr>
+    	<tr>
+        	<td>Cute hedgehogs</td>
+      		<td className="text-center"><span className="text-red text-lg">✖</span></td>
+        	<td className="text-center"><span className="text-green text-lg">✔</span></td>
+        	<td className="text-center">Joy</td>
+   		</tr>
+	</tbody>
+</table>
+</div>
+
+#### Key differences
+
+1. **Analytics features:** While LogRocket includes some product analytics features, it lacks important data analysis features like support for custom formulas, group analytics, and a SQL query builder that are built into PostHog.
+
+2. **Heat maps and scroll maps:** PostHog supports click maps, but LogRocket offers more in-depth heat maps and scroll depth visualizations, which are ideal for UX research and click-through optimization.
+
+3. **Pricing & free tier:** LogRocket's free tier is limited to just 1,000 sessions per month and excludes numerous features, such as product analytics. In contrast, all PostHog users get 1 million events and 15k session recordings for free every month.
```

Comment:
What about PostHog features?

Reply:
As in, what features are included for free? I'm unclear what you mean.

Reply:
Maybe say "you can get all the features of PostHog for free" or something like that.

Reply:
Have added the line "and all the core features" on the end